plugins/kubernetes/test/controllers/kubernetes/namespaces_controller_test.rb

```diff
@@ -135,7 +135,7 @@
 
       it "shows creation errors" do
         assert_request(:get, "http://foobar.server/api/v1/namespaces/test", to_return: {status: 404}) do
-          assert_request(:post, "http://foobar.server/api/v1/namespaces", to_timeout: [], times: 4) do
+          assert_request(:post, "http://foobar.server/api/v1/namespaces", to_timeout: [], times: 6) do
```

Comment:
I'd extract 5 and 6 to constants to improve clarity and to reduce further changes, if we want to change from 5 to 10.

Reply:
https://github.com/zendesk/samson/pull/3395